controllers/collection.lua

```diff
@@ -87,53 +78,106 @@ CollectionController.GET.user_collections = function (self)
     end
 end
 
-CollectionController.GET.collection = function (self)
+CollectionController.GET.collections = function (self)
     -- GET /users/:username/collections/:collection_slug
     -- Description: Get info about a collection.
     -- Parameters:  username, collection_slug, ...
 
-    return -- TODO
-    -- local collection = assert_collection_exists(self)
-    -- local project_count = collection:count_projects
-    -- collection.projects_count = project_count
-    -- return jsonResponse(collection)
+    local collection = assert_collection_exists(self)
+    collection.projects_count = collection:count_projects()
+    return jsonResponse(collection)
 end
 
-CollectionController.POST.collection = function (self)
+CollectionController.POST.collections = json_params(function (self)
     -- POST /users/:username/collections/:collection_slug
```

Comment:
Same here, this saves a single collection. I'd rather name functions by their functionality, not by their URL, so that you know what they're returning or doing when you call them from `api.lua`.